gaffer-as-a-service/gaas-rest/pom.xml

```diff
@@ -98,12 +88,6 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-api</artifactId>
-            <version>${junit.version}</version>
```

Comment:
Should be in parent so version not needed

Reply:
done